test/kuzzle/log.test.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const sinon = require('sinon');
+const should = require('should');
+
+const Logger = require('../../lib/kuzzle/log');
+
+describe('/lib/kuzzle/log', () => {
+  let logger;
+
+  beforeEach(() => {
+    global.kuzzle = {
+      once: sinon.stub(),
+      id: 'nasty-author-4242',
+      emit: sinon.stub(),
+      asyncStore: {
+        exists: () => false,
+        has: () => true,
+        get: () => ({ id: 'request-unique-id' })
+      }
+    };
```

Comment:
(nitpicking) ...or you could simply instantiate a new KuzzleMock object, it should take care of everything (apart from the asyncStore, I don't recall having that mocked)